pkg/local_object_storage/metabase/delete.go

```diff
@@ -60,11 +61,22 @@ func (db *DB) deleteGroup(tx *bbolt.Tx, addrs []oid.Address) error {
 	refCounter := make(referenceCounter, len(addrs))
 	currEpoch := db.epochState.CurrentEpoch()
 
+	var rawDeleted uint64
 	for i := range addrs {
-		err := db.delete(tx, addrs[i], refCounter, currEpoch)
+		removed, err := db.delete(tx, addrs[i], refCounter, currEpoch)
 		if err != nil {
 			return err // maybe log and continue?
 		}
+
+		if removed {
+			rawDeleted++
+		}
+	}
+
+	err := db.updateCounter(tx, rawDeleted, false)
+	if err != nil {
+		db.log.Error("could not decrease object counter",
+			zap.Error(err))
```

Comment:
No, let's not log anything inside of a transaction. It is potentially a blocking call.

Reply:
missed that it is inside a bolt TX

return back the returned code: no logging, direct err to a caller only